src/popup/router/components/Header.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <div
     v-if="showNavigation && !aeppPopup"
-    class="header"
-    :class="{ 'not-logged-in': !isLoggedIn }"
+    :class="['header', { 'not-logged-in': !isLoggedIn, 'new-ui': $route.meta.newUI }]"
```

Comment:
I don't see any reason to combine the static CSS class names and the dynamic ones.

Reply:
it's better than we have duplicated property

Reply:
Let's agree to disagree as this is a subjective thing. Static class is not processed by the vue internals. Dynamic class needs to be processed. Avoiding unnecessary work in my opinion is a good thing.

Another thing is the situation where we have more static classes. It would be pain in the ass to write those as an array.

After all I'm leaving the decision to you.